ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts

```diff
@@ -198,6 +209,10 @@ export class SummaryComponent implements OnInit {
           };
         }
       });
+
+    // debug
+    results['debug'] = this.form.get('debug').value;
```

Comment:
I think we lost the debug option in the CLI command. Not able to see it. And now I think we should update the CLI command on debug option selection.